src/pybind/mgr/rbd_support/schedule.py

```diff
@@ -304,13 +306,13 @@ class Schedule:
 
     def __init__(self, name: str) -> None:
         self.name = name
-        self.items: Set[Tuple[Interval, Optional[StartTime]]] = set()
+        self.items: Set[Tuple[Optional[Interval], Optional[StartTime]]] = set()
```

Comment:
I don't think it is a good idea to store items with None interval in the items set. Why do we need this?

Reply:
I agree! I added this to satisfy mypy ; will try to find a way.